### ISSUE TITLE
Reported vulnerability in dependencies

### ISSUE THREAD

**Issue by rhwood** [chrisjshull/homebridge-nest#82]
**Describe the bug**

When I include this project as dependency in my project, `npm audit` reports:

```
                       === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.5                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ homebridge-nest                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ homebridge-nest > request-promise > lodash                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/577                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

This is with version 2.1.4 of this project (here is part of `npm list` on my project):

**Comment by JacVid** [comment]
Hello,

here is the messages I get:

> /homebridge # npm audit
> ```
>                    === npm audit security report ===                        
>                                                                             
>                                                                             
>                              Manual Review                                  
>          Some vulnerabilities require your attention to resolve             
>                                                                             
>       Visit https://go.npm.me/audit-guide for additional guidance           
> ```
> Moderate        Prototype Pollution
> Package         lodash
> Patched in      >=4.17.11
> Dependency of   homebridge-nest
> Path            homebridge-nest > request-promise > lodash
> More info       https://npmjs.com/advisories/782
> Low             Prototype Pollution
> Package         lodash
> Patched in      >=4.17.5
> Dependency of   homebridge-nest
> Path            homebridge-nest > request-promise > lodash
> More info       https://npmjs.com/advisories/577
> found 2 vulnerabilities (1 low, 1 moderate) in 2381 scanned packages
> 2 vulnerabilities require manual review. See the full report for details.

I've tried this:

```
$ npm i -g npm
$ npm i --save lodash.update
```
and even after a reboot, I still have a "lodash issue"

Any idea ?

**Comment by JacVid** [comment]
@oznu said:

It's a nested dependency, and I would say the risk is very low, but the reason that version keeps getting installed it's defined in the plugins `package-lock.json`:

https://github.com/chrisjshull/homebridge-nest/blob/master/package-lock.json#L622

An `npm update` / `npm audit fix` on that plugin should be all that is required.

@rhwood, can you please do it ?

It will be highly appreciated.

**Comment by rhwood** [comment]
Changes are in #84 

A complete `npm update` would require other changes due to breaking changes in firebase, so that was not done. Note that `npm audit fix` was unable to fix this due to fixing change in request-promise being in a version that did not match the allowed versions in _package.json_.

**Comment by JacVid** [comment]
Thanks for the answer. 

But as I'm a Newbie, I really don't understand what I should do when I look at this #84.

Do I have to do something on my side, or do I have to wait for a coming update ?

**Comment by rhwood** [comment]
Fixed by #84 

@JacVid if you update your working copy from the master branch of this repo, your working copy will contain the fix for this.
